### PR TITLE
Quick fix to update email duplicate bug

### PR DESCRIPTION
**Pull request by Ty-Won**
### Tickets:
Refer to ticket #622 

### List of changes:
Changed the updateAccount middleware to search account with req.body.email instead of account.email

## Type of change
-   [x] Bug fix (non-breaking change which fixes an issue)

### How has this been tested?
Manually with postman

### CODE REVIEW

**Comment by loreina** [comment]
@Ty-Won can you also update the changelog?

File: .github/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Changed
```
Comment (pierreTklein) [inline .github/CHANGELOG.md]: Nit, put in ### fixed

File: tests/account.test.js [hunk 2]
```diff
@@ -387,6 +388,29 @@ describe("PATCH update account", function() {
                 });
         });
     });
+
+    // fail due to attempt to update account email to one that already exists in DB
```
Comment (pierreTklein) [inline tests/account.test.js]: Is there a success test?

Comment (loreina) [reply tests/account.test.js]: @Ty-Won

Comment (Ty-Won) [reply tests/account.test.js]: Failure test, I'll write a success one!

**Pull request merged**